server/src/main/java/io/crate/execution/engine/indexing/GroupRowsByShard.java

```diff
@@ -105,13 +126,16 @@ public GroupRowsByShard(ClusterService clusterService,
              UpsertResultContext.forRowCount());
     }
 
+    /**
+     * BiConsumer is needed for compatibility of the grouper with BatchIterators.partition
+     */
     @Override
     public void accept(ShardedRequests<TReq, TItem> shardedRequests, Row row) {
-        apply(shardedRequests, row);
+        accept(shardedRequests, row, false);
```

Comment:
By default false (insert into swallow errors) but once we introduce a setting in https://github.com/crate/crate/issues/12218, we can pass that setting value here